app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java

```diff
@@ -72,6 +96,11 @@ public View getChildView(int groupPosition, final int childPosition,
         return convertView;
     }
 
+    /**
+     * returns total children count.
```

Comment:
and also I didn't understand what do you mean my "Do not add commits for each file you change" you mean squash them?